google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -2144,32 +2145,19 @@ def self.new_reference project_id, dataset_id, service
         #
         # @!group Data
         #
-        def insert table_id, rows, insert_ids: nil, skip_invalid: nil, ignore_unknown: nil, autocreate: nil
+        def insert table_id, rows, insert_ids: nil, skip_invalid: nil, ignore_unknown: nil, autocreate: nil, &block
```

Comment:
I know this PR does not add the yielded block, but it appears to me that the yield is undocumented.

Reply:
I think you are correct.

Reply:
Added docs for this. @quartzmo please re-review.

Reply:
Thanks!